manual/sphinx/user_docs/bout_options.rst

```diff
@@ -303,6 +314,34 @@ given, ``NXPE`` takes precedence and ``NYPE`` is ignored):
 
     NYPE = 1  # Set number of Y processors
 
+When choosing ``NXPE`` or ``NYPE``, they must also obey some constraints:
+
+- ``NXPE`` must be a factor of the number of grid points in the x-direction
+
+  - That is, ``(nx - 4) / NXPE`` must be an integer, assuming the usual two
+    boundary points
+
+- For simple slab grids, ``NYPE`` must be a factor of the number of grid points
```

Comment:
I think it would be easier to read if it wouldn't say "For simple slab grids" - because it is true for all cases.

Reply:
I agree